### PR TITLE
Fail fast in the specs when running from a path with special characters

### PR DESCRIPTION
**Pull request by segiddins**
\c @indirect

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -17,6 +17,10 @@
   abort "Run rake spec:deps to install development dependencies"
 end
 
+if File.expand_path(__FILE__) =~ %r{[^\w/\.]}
+  abort "The bundler specs cannot be run from a path that contains special characters"
```
Comment (hmistry) [inline spec/spec_helper.rb]: Nice and short. I suggest being more specific what special characters we're referring to. In this case we're looking for white spaces so lets say that as they can take action without having to looking it up. If down the road the list gets bigger, we can state the common ones and users can check the docs for a full list. I appreciate the quick fix!

Comment (indirect) [inline spec/spec_helper.rb]: 👍

Comment (segiddins) [inline spec/spec_helper.rb]: It now prints the offending character

Comment (hmistry) [inline spec/spec_helper.rb]: Nice... really nice!

**Comment by indirect** [comment]
@homu r+

**Pull request merged**